examples/StockDonor/src/api/operations.graphql

```diff
@@ -1,21 +1,19 @@
-query pluginData($storeId: String!, $stockLineId: String!) {
+query pluginData($storeId: String!, $stockLineIds: [String!]) {
   pluginData(
     storeId: $storeId
     type: STOCK_LINE
-    filter: { relatedRecordId: { equalTo: $stockLineId } }
+    filter: { relatedRecordId: { equalAny: $stockLineIds } }
```

Comment:
I think should add `pluginName` filter here. If I want to add two stock line columns, this one should be filtered on `pluginName: { equalTo: 'StockDonor' }`, per the insert/update, while the second column would be filtering separately.

Almost makes me wonder if we should require pluginName filter in these plugin data queries, but I guess good to leave flexible for whatever user wants to do. But good to put in here, I missed it and was concerned that my stockDonor data was showing up in my new unrelated column!